ansible/roles_ocp_workloads/ocp4_workload_app_deploy_ilt/readme.adoc

```diff
@@ -8,17 +8,16 @@
 *** Debug task will print out: `pre_workload Tasks completed successfully.`
 
 ** Playbook: link:./tasks/workload.yml[workload.yml] - Used to configure the cluster
-*** This role creates a route for the image registry, sets up build defaults and removes self-provisioner from all users, installs Gogs.
+*** This role installs Gogs.
 *** Debug task will print out: `workload Tasks completed successfully.`
 
 ** Playbook: link:./tasks/post_workload.yml[post_workload.yml] - Used to
  configure the workload after deployment
 *** This role doesn't do anything here
 *** Debug task will print out: `post_workload Tasks completed successfully.`
 
-** Playbook: link:./tasks/remove_workload.yml[remove_workload.yml] - Used to
- delete the workload
-*** This role removes the production customizations
+** Playbook: link:./tasks/remove_workload.yml[remove_workload.yml] - Used to delete the workload
```

Comment:
This is a tasks file, it is not a playbook. This is wrong and confusing in a bunch of our readmes.

Reply:
We could do a global replace on all the readme.adocs

Reply:
Opened a PR to do just that :smile: